js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1442,20 +1444,25 @@ const EditingController = modules.ViewController.inherit((function() {
         _processCanceledEditingCell: function() { },
 
         _prepareEditCell: function(params, item, editColumnIndex, editRowIndex) {
-            const that = this;
-
             if(!item.isNewRow) {
                 params.key = item.key;
             }
 
-            if(that._isEditingStart(params)) {
+            if(this._isEditingStart(params)) {
                 return false;
             }
 
-            that._pageIndex = that._dataController.pageIndex();
+            this._pageIndex = this._dataController.pageIndex();
+
+            this._setEditRowKey(item.key);
+            this._setEditColumnNameByIndex(editColumnIndex);
 
-            that._setEditRowKey(item.key);
-            that._setEditColumnNameByIndex(editColumnIndex);
+            if(!params.column.showEditorAlways) {
```

Comment:
Why we need this condition?

Reply:
This is for the boolean or any showEditorAlways column where the state is added before calling the _prepareEditCell.